examples/deep_sleep.rs

```diff
@@ -0,0 +1,70 @@
+//! Tests deep sleep
+//!
+//! Enables multiple deep sleep wakeup sources and then enter deep sleep.
+//! There is no loop here, since the program will not continue after deep sleep.
```

Comment:
will it is implicit here, we could state - "There is no loop here, since the program will not continue after deep sleep. It always starts from the beginning after a deep sleep wake-up" To run the point absolutely clear to everybody